cabal-install/src/Distribution/Client/Init/Format.hs

```diff
@@ -76,6 +76,10 @@ fieldD fieldName fieldContents fieldComments includeField opts
         -- If the "--minimal" flag is set, strip comments.
         in fieldSWithContents comments
   where
+    isCabalVersion fn
```

Comment:
I stared at the call site for a while thinking it was a `_ -> Bool` function. What about `commentPositionFor`?